docs/deploying-to-azure-containerapps.md

```diff
@@ -730,6 +731,72 @@ az staticwebapp secrets list  \
   --output table
 ```
 
+# Load Testing
+
+Now time to add some load to the application.
+This way, we will be able to see the auto-scaling in Azure Container Apps.
+
+To add some load to an application, you can do it locally using [JMeter](https://jmeter.apache.org), but you can also do it remotely on Azure using [Azure Load Testing](https://azure.microsoft.com/services/load-testing) and JMeter.
+Azure Load Testing is a fully managed load-testing service built for Azure that makes it easy to generate high-scale load and identify app performance bottlenecks.
+It is available on the [Azure Marketplace](https://azuremarketplace.microsoft.com).
+For that, we need to go the 
+
+To use Azure Load Testing, go to the [Azure Portal](https://portal.azure.com), search for the Marketplace and look for "_Azure Load Testing_"  in the Marketplace.
+Click on "_Create_":
+
+![load-testing-1-marketplace](../images/load-testing-1-marketplace.png)
+
+Create a load testing resource by giving it a name (`super-heroes-load-testing`), a location, and a resource group.
+Click on "Create":
+
+![load-testing-2-create](../images/load-testing-2-create.png)
+
+Creating a load testing resource can take a few moment.
+Once created, you should see the Azure Load Testing available in your resource group: 
+
+![load-testing-3-list](../images/load-testing-3-list.png)
+
+Select `super-heroes-load-testing` and click on "_Tests_" and then "Create".
+You can either create a quick load test using a wizard, or create a load test using a JMeter script.
+Choose this second option:
+
+![load-testing-4-upload-jmeter](../images/load-testing-4-upload-jmeter.png)
+
+Before uploading a JMeter script, create a load test by entering a name (eg. "_Make them fight_"), a description and click next: 
+
+![load-testing-5-create-test-1](../images/load-testing-5-create-test-1.png)
```

Comment:
This image has a spelling mistake.

In the `description` it says `Makes a super hero and sper villain fight`. It should be `Makes a super hero and super villain fight`